src/Microsoft.AspNet.Mvc.ModelBinding/Resources.resx

```diff
@@ -123,6 +123,9 @@
   <data name="ArgumentCannotBeNullOrEmpty" xml:space="preserve">
     <value>Value cannot be null or empty.</value>
   </data>
+  <data name="KeyValuePair_BothKeyAndValueMustBePresent" xml:space="preserve">
+    <value>A value is required.</value>
```

Comment:
The key should be self explanatory... I was not happy with this one though. Suggestions?